src/client/components/StudyGroupLogo/StudyGroupLogo.styles.css

```diff
@@ -0,0 +1,20 @@
+* {
```

Comment:
Do not write global styles in this file, styles should be only for _this_ component.

Reply:
yes I removed this